torch/onnx/symbolic_opset10.py

```diff
@@ -178,3 +179,67 @@ def flip(g, input, dims):
 
 def fmod(g, input, other):
     return g.op("Mod", input, other, fmod_i=1)
+
+
+@parse_args('v', 'v', 'v', 'i', 'i', 'i', 'v', 'i')
+def embedding_bag(g,
+                  embedding_matrix,
+                  indices,
+                  offsets,
+                  scale_grad_by_freq,
+                  mode,
+                  sparse,
+                  per_sample_weights,
+                  include_last_offset):
+    from torch.onnx.symbolic_opset9 import size, div
+
+    if not sym_help._is_none(per_sample_weights):
+        return sym_help._onnx_unsupported('embedding_bag  with per_sample_weights')
+
+    if offsets.type().sizes() is not None:
+        if indices.node().kind() != 'prim::Param':
+            embeddings = g.op("Gather", embedding_matrix, indices)
+            dim_0 = size(g, offsets, g.op("Constant", value_t=torch.LongTensor([0])))
+            dim_1 = div(g, size(g, indices, g.op("Constant", value_t=torch.LongTensor([0]))),
+                        size(g, offsets, g.op("Constant", value_t=torch.LongTensor([0]))))
+            dim_2 = size(g, embedding_matrix, g.op("Constant", value_t=torch.LongTensor([1])))
+
+            shape = [dim_0, dim_1, dim_2]
+            shape = g.op("Concat", *shape, axis_i=0)
+
+            embeddings = g.op("Reshape", embeddings, shape)
+            if mode == 0:
+                embeddings = g.op("ReduceSum", embeddings, axes_i=[1], keepdims_i=0)
+            elif mode == 1:
+                embeddings = g.op("ReduceMean", embeddings, axes_i=[1], keepdims_i=0)
+            else:
+                embeddings = g.op("ReduceMax", embeddings, axes_i=[1], keepdims_i=0)
+            return embeddings, None, None, None
```

Comment:
do you know why there are four outputs for the `embedding_bag` schema?

Reply:
I added a comment explaining that the three last outputs are unused in torch.nn modules.